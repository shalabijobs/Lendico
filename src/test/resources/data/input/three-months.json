{
  "loanAmount": "5000",
  "nominalRate": "5.0",
  "duration": 3,
  "startDate": "2018-01-01T00:00:01Z"
}
