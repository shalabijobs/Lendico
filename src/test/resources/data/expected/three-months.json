[
  {
    "borrowerPaymentAmount": 1680.57,
    "date": "2018-01-01T00:00:01",
    "initialOutstandingPrincipal": 5000.0,
    "interest": 20.83,
    "principal": 1659.74,
    "remainingOutstandingPrincipal": 3340.26
  },
  {
    "borrowerPaymentAmount": 1680.57,
    "date": "2018-02-01T00:00:01",
    "initialOutstandingPrincipal": 3340.26,
    "interest": 13.92,
    "principal": 1666.65,
    "remainingOutstandingPrincipal": 1673.61
  },
  {
    "borrowerPaymentAmount": 1680.58,
    "date": "2018-03-01T00:00:01",
    "initialOutstandingPrincipal": 1673.61,
    "interest": 6.97,
    "principal": 1673.61,
    "remainingOutstandingPrincipal": 0.0
  }
]
